corehq/form_processor/backends/sql/update_strategy.py

```diff
@@ -111,6 +111,9 @@ def _apply_update_action(self, update_action):
             if key == 'location_id':
                 # special treatment of location_id
                 self.case.location_id = value
+            elif key == 'name':
+                # replicate legacy behaviour
+                self.case.name = value
```

Comment:
yea, this is a bit odd. The real case name property is called `case_name` so technically if you set the `name` property then you just get a dynamic case property called `name`.

I'm not sure if the old functionality was intentional but it works because we just set dynamic case props as attributes on the couch case object so setting `name` is the same as changing the case name property.

Either way I figured if a user is setting a property called `name` they are likely intending to set the case name.